for_dict_challenges.py

```diff
@@ -12,8 +12,16 @@
     {'first_name': 'Маша'},
     {'first_name': 'Петя'},
 ]
+names = dict()
+for student in students:
+    if student['first_name'] not in names:
+        names[student['first_name']] = 1
+    else:
+        names[student['first_name']] += 1
```

Comment:
Как продвинутый вариант можно использовать get с аргументом по умолчанию 0:
```suggestion
    names[student['first_name']] = names.get(student['first_name'], 0) + 1 
```